src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/node.type.ts

```diff
@@ -1,20 +1,33 @@
 export interface TerminalViewProps {
-  nodeName: string;
-  shell: any;
-  containerName: string;
-  socketConnection: SocketConnectionType;
-  terminalCleared: boolean;
-  setTerminalCleared: (terminalCleared: boolean) => void;
-  setSocketConnection: (socketConnection: SocketConnectionType) => void;
+    nodeName: string
+    shell: any
+    containerName: string
+    socketConnection: SocketConnectionType
+    terminalCleared: boolean
+    setTerminalCleared: (terminalCleared: boolean) => void
+    setSocketConnection: (socketConnection: SocketConnectionType) => void
+    clusterTerminal?: boolean
+    terminalId?: string
+    fetchRetry?: boolean
```

Comment:
Can we rename`fetchRetry` to `isFetchRetry`, `clusterTerminal ` to `isClusterTerminal`, `terminalCleared` to `isTerminalCleared`, `toggleOption` to `isToggleOption`,